drivers/sdhc/sdhc_esp32.c

```diff
@@ -1195,7 +1187,7 @@ static int sdhc_esp32_request(const struct device *dev, struct sdhc_command *cmd
 	}
 
 	if ((ret_esp != 0) || esp_cmd.error) {
-		LOG_DBG("\nError for command: %u arg %08x ret_esp = 0x%x error = 0x%x\n",
+		LOG_DBG("Error command: %u arg %08x ret_esp = 0x%x error = 0x%x\n",
```

Comment:
Check compliance.

Reply:
Is this a legit warning? Running locally or checking the Summary we see no errors.